### ISSUE TITLE
jsx-wrap-multiline + React 16 fragments

### ISSUE THREAD

**Issue by johnsoft** [palantir/tslint-react#118]
React 16 allows you to return an array from a render function, which brings up a new pattern that's possibly worth supporting. The rule flags the first div in this code as an error due to missing parentheses:

```ts
class MyComponent extends React.Component {
  public render() {
    return [
      <div>
        <SomeCrazyLongComponent />
      </div>,
      <div>
        <OtherCrazyLongComponent />
      </div>,
    ];
  }
}
````

The suggested fix wastes a lot of space and doesn't gain anything readability-wise:

```ts
class MyComponent extends React.Component {
  public render() {
    return [
      (
        <div>
          <SomeCrazyLongComponent />
        </div>
      ),
      (
        <div>
          <OtherCrazyLongComponent />
        </div>
      ),
    ];
  }
}
````

What do you think about changing the rule to support cases like this? My suggested criteria is that the opening and closing (if present) angle brackets starting in the same column should cause the code to be OK under this rule. From my brief glance at [the tests](https://github.com/palantir/tslint-react/blob/master/test/rules/jsx-wrap-multiline/test.tsx.lint), all the cases would stay the same under this change.

**Comment by adidahiya** [comment]
This is basically the same as https://github.com/palantir/tslint-react/issues/79#issuecomment-289923337 and I would absolutely accept a PR to make the rule more lenient here.

**Comment by johnsoft** [comment]
Whoops sorry, I tried searching but clearly I didn't search well enough. I'll take the liberty of closing this as a dup
